backend/tests/test_services.py

```
@@ -1,839 +1,848 @@
-        
-#         # Test with no last_seen_post
-#         mock_get_newer_posts.reset_mock()
-#         mock_get_newer_posts(None, db)
-#         mock_get_newer_posts.assert_called_once_with(None, db)
-        
-#         # Test with non-existent post
-#         mock_get_newer_posts.reset_mock()
-#         mock_get_newer_posts(999, db)
-#         mock_get_newer_posts.assert_called_once_with(999, db)
+# Test for get_newer_posts function
+def test_get_newer_posts():
```

Ruff (0.8.2)

Missing return type annotation for public function `test_get_newer_posts`

Add return type annotation: `None`

(ANN201)